test/test_pty.rs

```diff
@@ -0,0 +1,36 @@
+use nix::pty::openpty;
+use nix::unistd::{read, write, close};
+
+#[test]
+fn test_openpty() {
+    // TODO: figure out the right termios settings to pass in here
```

Comment:
Fixing it in `libc` won't fix it because as of right now `Termios` is directly defined in `nix/sys/termios.rs`. If this lands before your PR, it'd make sense to fix it afterwards